geometry_msgs/Pose hand_goal
bool load_pc
float32[3] pos_gains
float32[3] rot_gains
float32[3] hand_offset
int32 mode